packages/mui-material/src/Menu/Menu.js

```diff
@@ -188,32 +217,42 @@ const Menu = React.forwardRef(function Menu(inProps, ref) {
       }}
       transformOrigin={isRtl ? RTL_ORIGIN : LTR_ORIGIN}
       slots={{
-        paper: PaperSlot,
         root: slots.root,
+        paper: PaperSlot,
+        backdrop: slots.backdrop,
+        transition: slots.transition,
       }}
       slotProps={{
         root: rootSlotProps,
         paper: paperSlotProps,
+        backdrop:
+          typeof slotProps.backdrop === 'function'
+            ? slotProps.backdrop(ownerState)
+            : slotProps.backdrop,
```

Comment:
Should we use `mergeSlotProps` here?

Reply:
No because it will produce a different behavior. The reason to check for typeof function is to use the `ownerState` of the Menu, not the Popover.